### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by LouisSzeto**
#### Description
Update landing page to contain information of Kraken brokerage.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [X] All new and existing tests passed.
- [X] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,247 @@
...
 
-Template Brokerage Plugin for [Lean](https://github.com/QuantConnect/Lean)
+## Introduction
 
-See the [brokerage development guide](https://www.quantconnect.com/tutorials/open-source/brokerage-development-guide)
\ No newline at end of file
+This repository hosts the Kraken Brokerage Plugin Integration with the QuantConnect LEAN Algorithmic Trading Engine. LEAN is a brokerage agnostic operating system for quantitative finance. Thanks to open-source plugins such as this [LEAN](https://github.com/QuantConnect/Lean) can route strategies to almost any market.
+
+[LEAN](https://github.com/QuantConnect/Lean) is maintained primarily by [QuantConnect](https://www.quantconnect.com), a US based technology company hosting a cloud algorithmic trading platform. QuantConnect has successfully hosted more than 200,000 live algorithms since 2015, and trades more than $1B volume per month.
+
```
Comment (DerekMelchin) [inline README.md:12]: Missing link to brokerage doc page

File: README.md [hunk 2]
```diff
@@ -1,7 +1,247 @@
...
+3.  Enter the number of the organization that has a subscription for the Kraken module.
+
+    ```
+    $ lean live "My Project" 
+
+    Select the organization with the Kraken module subscription:
+    1. Organization 1
+    2. Organization 2
+    3. Organization 3
+       Enter an option:
```
Comment (DerekMelchin) [inline README.md:69]: Remove indentation of last line

File: README.md [hunk 3]
```diff
@@ -1,7 +1,247 @@
...
+### QuantConnect Cloud
+
+  This plugin is integrated in the QuantConnect Cloud Platform where you can use this integration with a simple visual interface, and harness the QuantConnect Live Data Feed. For most users this is substantially cheaper and easier than self-hosting. For more information see the [Kraken documentation page](https://www.quantconnect.com/docs/v2/our-platform/live-trading/brokerages/kraken). 
+  
+### Locally
+
+Follow these steps to start local live trading with the Kraken brokerage:
+
+1.  Open a terminal in your [CLI root directory](https://www.quantconnect.com/docs/v2/lean-cli/initialization/directory-structure#02-lean-init).
+2.  Run lean live "`<projectName>`" to start a live deployment wizard for the project in ./`<projectName>` and then enter the brokerage number.
```
Comment (DerekMelchin) [inline README.md:37]: lean live "`<projectName>`" => `lean live "<projectName>"`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,247 @@
...
+    8. Samco
+    9. Terminal Link
+    10. Trading Technologies
+    11. Kraken
+    12. FTX
+    13. IQFeed
+    14. Polygon Data Feed
+    15. Custom Data Only
+  
+        To enter multiple options, separate them with comma:
```
Comment (DerekMelchin) [inline README.md:110]: Remove indentation of last line

File: README.md [hunk 5]
```diff
@@ -1,7 +1,247 @@
...
+    ```
+
+7. View the result in the `<projectName>`/live/`<timestamp>` directory. Results are stored in real-time in JSON format. You can save results to a different directory by providing the --output `<path>` option in step 2.
```
Comment (DerekMelchin) [inline README.md:113]: `<projectName>`/live/`<timestamp>` => `<projectName>/live/<timestamp>`

--output `<path>` => `--output <path>`

File: README.md [hunk 6]
```diff
@@ -1,7 +1,247 @@
...
+    10. Trading Technologies
+    11. Kraken
+    12. FTX
+    13. IQFeed
+    14. Polygon Data Feed
+    15. Custom Data Only
+  
+        To enter multiple options, separate them with comma:
+    ```
+
```
Comment (DerekMelchin) [inline README.md:112]: Missing "If you chose IQFeed... " text

File: README.md [hunk 7]
```diff
@@ -1,7 +1,247 @@
...
+- Limit-If-Touch Order
+- Stop-Market Order
+- Stop-Limit Order
+
+
+## Downloading Data
+
+For local deployment, the algorithm needs to download the following dataset:
+
+[Kraken Crypto Price Data](https://www.quantconnect.com/datasets/kraken-crypto-price-data) provided by CoinAPI
```
Comment (DerekMelchin) [inline README.md:137]: Missing `-` to make it a list

**Review by DerekMelchin — DISMISSED** [review]

**Review by AlexCatarino — APPROVED** [review]

**Pull request merged**